### ISSUE TITLE
Revealed passwords are difficult to read due to ambiguous glyph appearances

### ISSUE THREAD

**Issue by wilt00** [mozilla-mobile/firefox-ios#12229]
## Steps to reproduce
* Navigate to password detail screen
* Reveal password

### Expected behavior
* It is easy to tell characters like lowercase l and uppercase I apart from on another

### Actual behavior
* Due to the sans-serif font used, these characters look very similar:

<img width="250px" src="https://user-images.githubusercontent.com/7883508/197928880-81597834-cb3d-4ca5-bcf0-cfba0a447544.png" />

I've tested the following change and would be happy to submit as a PR:

```diff
--- a/Client/Frontend/Settings/LoginDetailViewController.swift
+++ b/Client/Frontend/Settings/LoginDetailViewController.swift
@@ -161,6 +161,8 @@ extension LoginDetailViewController: UITableViewDataSource {
             loginCell.descriptionLabel.text = login.decryptedPassword
             loginCell.descriptionLabel.returnKeyType = .default
+            loginCell.descriptionLabel.font = UIFont.monospacedSystemFont(ofSize: 16, weight: .regular)
             loginCell.displayDescriptionAsPassword = true
             loginCell.isEditingFieldData = isEditingFieldData
```

Which looks like:

<img width="300px" alt="Screen Shot 2022-10-25 at 8 32 53 PM" src="https://user-images.githubusercontent.com/7883508/197929431-625d09c0-a413-49b9-80aa-e5f89b206172.png">

Although it might be better to create this font as a constant in `LoginDetailTableViewCell` alongside the existing fonts and have that class take a parameter, please advise.

### Device & build information
* Device: iPhone XR
* Build version: d42f93724df509104bd3cc481bb780c10f181560
### Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5155)

**Comment by cwzilla** [comment]
Spoke with the design systems team and we'd like to utilize an SF mono font and improve the password field. Not a high priority, can we add this to the list of UX debt to-dos?

**Comment by dveditz** [comment]
> Spoke with the design systems team and we'd like to utilize an SF mono font 

I'm happy with the readability of (most?) mono fonts, but that will exacerbate _my_ bugbear: long passwords are truncated with no indication they are truncated and no way to see the full version. The "copy" button still works, but I'd really really prefer not putting my passwords in the clipboard where every other app can get at them.

**Comment by dveditz** [comment]
[never mind if you got mail about a comment I made. I have that problem on the Android version, not the iOS version]

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v115 (31834) with iPhone 13 Pro (15.7.1).
